oscarapi/serializers/basket.py

```diff
@@ -92,9 +92,9 @@ class BasketLineSerializer(OscarHyperlinkedModelSerializer):
     url = DrillDownHyperlinkedIdentityField(
         view_name='basket-line-detail',
         extra_url_kwargs={'basket_pk': 'basket.id'})
-
     attributes = LineAttributeSerializer(
-        many=True, fields=('url', 'option', 'value'), required=False)
+        many=True, fields=('url', 'option', 'value'),
```

Comment:
What exactly do you mean by many? is 3 many? is 100 many? Or maybe 6 trillion? Please specify more accurately.